### DISCUSSION TITLE
Convert from MP3

### DISCUSSION THREAD

**Discussion by kilmerc** [openaudible/openaudible#901] [Q&A] [answered]
Love the software, keep it up. When I right-click on a file that is MP3 for example the convert to M4B button is greyed out. What are the parameters that cause a file to be able to be converted or not?

**Comment by openaudible** [comment, marked as answer]
Thanks for the kind words!

If you have an MP3 file, we don't currently support converting to M4B, as you'll lose audio quality which we want to avoid. 

We recommend keeping it mp3 format-- won't hurt to have some audiobooks in mp3 and some in m4b.. (although it isn't as tidy.) we feel adding that feature could cause people to lose audio quality and gain no real value.

You can convert from audible files to either mp3 or m4b. And I think you can convert m4b to mp3 (for instance, to get the file smaller at lower quality) but not positive about that.

**Reply by kilmerc** [reply]
Thank you! I didn't know most of that. Very helpful.
